CorsaSchoolManagementSystem/Views/studentDetails.php

```diff
@@ -23,139 +29,303 @@
+        </div>
+      </div>
+
+      <div class="tab-pane fade" id="parentInformation" role="tabpanel" aria-labelledby="parentInformation-tab">
+        <div class="row">
+            <div class="col-4">
+                <ul style="list-style: none;" >
+                    <li class="my-2"> <strong>FATHER'S NAME : </strong> <?php echo $studentDetails->fathersName?>  </li>  
+                    <li class="my-2"> <strong>TEL. NUMBER : </strong> <?php echo $studentDetails->fathersTelephone?>  </li>  
+                    <li class="my-2"> <strong>OCCUPATION : </strong> <?php echo $studentDetails->fathersOccupation?></li>  
+                    <li class="my-2"> <strong>HOMETOWN : </strong> <?php echo $studentDetails->fathersHometown?> </li>  
+                </ul>
+            </div>
+            <div class="col-4">
+                <ul style="list-style: none;" >
+                    <li class="my-2"> <strong>MOTHER'S NAME : </strong> <?php echo $studentDetails->mothersName?> </li>  
+                    <li class="my-2"> <strong>TEL. NUMBER : </strong>  <?php echo $studentDetails->mothersTelephone?></li>  
+                    <li class="my-2"> <strong>OCCUPATION : </strong> <?php echo $studentDetails->mothersOccupation?></li>  
+                    <li class="my-2"> <strong>HOMETOWN : </strong> <?php echo $studentDetails->mothersHometown?></li>  
+                </ul>
+            </div>
+        </div>
+      </div>
     </div>
+    
   </div>
-  <div class="tab-pane fade" id="academicInformation" role="tabpanel" aria-labelledby="academicInformation-tab">
-    <center><h4>Terminal Report</h4></center>
-    <h1>content</h1>
-  </div>
-  <div class="tab-pane fade" id="parentInformation" role="tabpanel" aria-labelledby="parentInformation-tab">
-    <div class="row">
-        <div class="col-4">
-            <ul style="list-style: none;" >
-                <li> <strong>FATHER'S NAME :</strong> Mr. Samuel Atikuni </li>  
-                <li> <strong>TEL. NUMBER :</strong>  03423231322</li>  
-                <li> <strong>OCCUPATION :</strong> 03423231322</li>  
-                <li> <strong>HOMETOWN :</strong> Sefwi Bekwai </li>  
-            </ul>
-        </div>
-        <div class="col-4">
-            <ul style="list-style: none;" >
-                <li> <strong>MOTHER'S NAME :</strong> Mr. Samuel Atikuni </li>  
-                <li> <strong>TEL. NUMBER :</strong>  03423231322</li>  
-                <li> <strong>OCCUPATION :</strong> 03423231322</li>  
-                <li> <strong>HOMETOWN :</strong> Sefwi Bekwai </li>  
-            </ul>
+    <!-- bills payment -->
+
+    <!-- Button trigger modal -->
+  <div class="container" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
+    <button type="button" class="btn btn-primary mt-5 mb-2 mx-5" data-bs-toggle="modal" data-bs-target="#staticBackdrop" style="box-shadow: 0 15px 20px 0 rgba(0, 0, 0, 0.1), 0 6px 15px 0 rgba(0, 0, 0, 0.14);">
```

Comment:
This button shows Terminal Report Modal...is it dummy information? You can update it to have the right information. 

<img width="1393" alt="Screenshot 2022-12-07 at 3 26 15 PM" src="https://user-images.githubusercontent.com/10617656/206221090-f56dd8e1-2c99-459f-9f7b-77cc71801ecd.png">